site/content/blog/2021-12-01-whats-new-in-svelte-december-2021.md

```diff
@@ -0,0 +1,91 @@
+---
+title: What's new in Svelte: December 2021
+description: "Svelte Summit Fall 2021 Recap, Rich Harris joins Vercel, SvelteKit opens GitHub Discussions"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+With SvelteKit getting more and more stable each day, there's not much to cover in terms of code changes other than bug fixes... So, in this month's newsletter, we'll be covering Svelte Summit Fall 2021!
+
+If you want to dive deep into the last month's worth of bug fixes, check out the [Svelte](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md) and [SvelteKit](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md) changelogs, respectively.
+
+
+## What happened at Svelte Summit?
+
+If you missed Svelte Summit, you can watch the entire live stream on [YouTube](https://www.youtube.com/watch?v=1Df-9EKvZr0) and catch a recap in the [#svelte-summit channel on Discord](https://discord.gg/YmHcdnhu).
```

Comment:
"and catch a recap in the channel on Discord" -> "or catch the recap in the Discord channel"